mscr-ui/src/common/utils/has-permission.tsx

```diff
@@ -97,13 +104,17 @@ export function checkPermission({
   actions,
   targetOrganizations,
 }: checkPermissionProps) {
+  console.log(targetOrganizations);
+
+  console.log("check content creation permission");
```

Comment:
Preferably remove or comment out console.logs. Although at least for our release build in the future we can probably put some code to disable console logging.

Reply:
removed